timeseries/tests/unittests/models/test_models.py

```diff
@@ -18,7 +18,7 @@
 from autogluon.timeseries.models.gluonts import GenericGluonTSModel
 
 from ..common import DUMMY_TS_DATAFRAME, dict_equal_primitive, get_data_frame_with_item_index
-from .test_gluonts import TESTABLE_MODELS as GLUONTS_TESTABLE_MODELS
+from .test_gluonts import TESTABLE_MX_MODELS as GLUONTS_TESTABLE_MODELS
```

Comment:
Why do we only test mxnet models here instead of all models (`TESTABLE_MODELS`)?

Reply:
pytorch tests coming soon.